### PR TITLE
Fixed font-size.

### PR DESCRIPTION
**Pull request by shubhkhanna**
* [x] Formal Education: A card with the name of the College, Degree, A picture or Illustration relevant to it,Activities
#14

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Adjust the size of cards, it is breaking out. We don't want a horizontal scroll balll

File: education.html [hunk 1]
```diff
@@ -17,15 +17,90 @@
     <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
     <ul class="menu">
       <li><a href="./index.html">Home</a></li>
-      <li><a href="./experience.html">Experience</a></li>
       <li><a href="#">About</a></li>
+      <li><a href="./experience.html">Experience</a></li>
```
Comment (sakshi1499) [inline education.html:21]: You shouldn't make such changes

File: css/education.css [hunk 2]
```diff
@@ -155,4 +155,149 @@ h5 {
       margin-top: 11rem;
       padding-left: 40px;
     }
-  }
```
Comment (sakshi1499) [inline css/education.css:158]: Please don't add or subtract lines. Such changes shouldn't be pushed. Its not a good practice.

**Comment by smaranjitghose** [comment]
Remove the changes to the Nav Bar and give the rest as @sakshi1499 suggests. @Adyasha8105 is working on a common nav bar for all

**Comment by shubhkhanna** [comment]
@smaranjitghose sir, please review it.

**Comment by smaranjitghose** [comment]
Fix the merge conflicts

**Comment by shubhkhanna** [comment]
I'm closing this PR due to navbar issue...looking forward to create new pull request. 
Closed #31

**Pull request closed without merging**